src/features/page_view_timing/aggregate/index.js

```diff
@@ -94,7 +94,7 @@ export class Aggregate extends AggregateBase {
       if (lcpEntry.url) {
         attrs['elUrl'] = cleanURL(lcpEntry.url)
       }
-      if (lcpEntry.element && lcpEntry.element.tagName) {
+      if (lcpEntry.element?.tagName) {
         attrs['elTag'] = lcpEntry.element.tagName
       }
```

Comment:
`onLCP` can trigger with just a value but empty entries `[]`. Rather than quitting, it's better to check if `lcpEntry` is `undefined`, and if so, skip over ln 89-99.

(I still don't know how that happens that isn't something wrong with their instrumentation. `onLCP` only verifiably trigger with empty entry on b/f cache restore.)

Reply:
No, skipping the entire callback for FID is fine, because it doesn't have a similar case of defined value w/ no entries in the src code. Unsure what we'd even report without an user-input entry to reference.